cmake/IconBuilder/main.cpp

```diff
@@ -39,10 +40,38 @@ int main(int argc, char* argv[])
 
   const std::vector<std::string> args{argv, argv + argc};
 
-  const auto& iconFormat = args.at(1);
-  const auto outputDir = File{args.at(2)};
-  const auto smallIconImageFile = args.at(3) == "<None>" ? File{} : File{args.at(3)};
-  const auto largeIconImageFile = args.at(4) == "<None>" ? File{} : File{args.at(4)};
+  using Version = std::tuple<int, int, int>;
+
+  const auto jucerVersion = [&args]() {
+    if (args.at(1) == "latest")
+    {
+      return Version{5, 4, 0};
```

Comment:
Should we extract "latest" to the top of this file ?

Reply:
I don't think that's valuable. When reading the code which compares `jucerVersion` to decide which function to call, you wouldn't know what `latest` is, until you jump to the top of the file. I prefer to keep that knowledge localized, even if it's a bit duplicated.